.vib/reportserver/goss/vars.yaml

```diff
@@ -0,0 +1,21 @@
+binaries:
+  - gosu
+  - java
+  - mysql
+  - render-template
+files:
+  - mode: "0664"
+    paths:
+      - /opt/bitnami/reportserver/WEB-INF/classes/persistence.properties
+      - /opt/bitnami/reportserver/WEB-INF/classes/reportserver.properties
+      - /opt/bitnami/reportserver/WEB-INF/classes/rsinit.properties
+      - /opt/bitnami/reportserver/WEB-INF/web.xml
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/tomcat/webapps
```

Comment:
There is something strange here. We are checking `/bitnami/tomcat/webapps`, but we are supposed to be creating a symlink in the postunpack stage via:
```
ln -s "$REPORTSERVER_BASE_DIR" "${BITNAMI_ROOT_DIR}/tomcat/webapps/reportserver"
```

The issue is that I can't find said symlink. Could this be an issue with out initialisation logic or is it expected? Have you noticed something amiss?

Reply:
It seems it's because we are not persisting the app as in other applications using `persist_app "$app_name" "DATA_TO_PERSIST"`.